src/manufacturer/manufacturerDialog.component.tsx

```diff
@@ -84,6 +84,7 @@ function ManufacturerDialog(props: ManufacturerDialogProps) {
     reset(initialManufacturer);
   }, [initialManufacturer, reset]);
 
+  // Clears form errors when a value has be changed
```

Comment:
```suggestion
  // Clears form errors when a value has been changed
```
I didn't event read my own suggestion properly 😄 sorry.